posthog/api/insight.py

```diff
@@ -270,7 +268,15 @@ def get_serializer_class(self) -> Type[serializers.BaseSerializer]:
         return super().get_serializer_class()
 
     def get_queryset(self) -> QuerySet:
-        queryset = super().get_queryset().prefetch_related("dashboards")
+        queryset = super().get_queryset()
+        queryset = queryset.prefetch_related("dashboards")
+        queryset = queryset.prefetch_related("dashboards__created_by")
+        queryset = queryset.select_related("created_by")
+        queryset = queryset.select_related("last_modified_by")
+        queryset = queryset.select_related("team")
+        queryset = queryset.select_related("dashboard__team")
+        queryset = queryset.select_related("dashboard__team__organization")
```

Comment:
Protip: you do `queryset = super().get_queryset().prefetch_related("foo", "foo_xyz").select_related("bar", "baz")` to amke this more concise ;)